src/components/Hero.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import skyline from '../images/skyline.svg'
+
+const Hero = () => {
+  return (
+    <section>
+      <div className="hero-content-container">
```

Comment:
No need for this div. The above `section` tag will allow us to style this container as needed.